clean-language/README-copy.md

```diff
@@ -0,0 +1,146 @@
+(Q) is there more about the radar system of the old skool watch
+(A) They are strong radars that have the power to bring forth other radars
+(Q) and whereabouts is that strong radars that have the power to bring forth other radars
+(A) its in my body
+(Q) is there more about the radar system of the old skool watch?
+(A) it is moving
+(Q) and the moving radar system, it is like what?
+(A) one big energy field
+(Q) what kind of energy field
+(A) an energy field that grows
+(Q) where is that energy field
+(A) all around me (and growing)
+(Q) then what happens
+(A) I move that energy towards others
+(Q) ...
+@@ Bridge with the next chapter.
+
+## The power of metaphor
+### Metaphors improve comprehension
+What is a metaphor? It is a figure of speech that describes an object or action in a way that isn’t literally true, but helps explain an idea or make a comparison.
+The best ones link something which is less understood to something which is more understood.
+As a result, good metaphors improve our comprehension of the ideas we might have not understood otherwise.
+Metaphors add colours to the story and make it "sticky".
+They enable us to show instead of tell.
+
+### Changing metaphors changes the way we think
+Did you know that on average we use six metaphors a minute? Surprised? Me too! They are so embedded into our speech that mostly we are unconscious about them.
+Research suggests that metaphors actually change the way we think of a concept on an unconscious level.
+In 2011 there was a study conducted in which two groups of readers have been presented with two articles describing the same crime.
+In the first article the crime was compared to a beast and in another to a disease.
+When asked to solve the crime issue, readers of the first article suggested control strategies while the second group came up with diagnostic or treatment strategies.
+So actually changing the metaphor changes how we think about the issue or an idea at hand.
+
+### Metaphors are capable of changing your experience
+In communication we often fail to see the connection the other person is making in their head.
+So we declare it nonsense.
+It's no wonder that we clash with each other, because we assign different meanings to the same things.
+ 
+In my clean language workshop it was an eye opening experience to observe a magnitude of metaphors people create in response to the same question.
+
+## Clean language and clean questions
+Clean Language is a questioning technique used for discovering, exploring and working with people's own personal metaphors.
+Clean means the questioner only uses the words of the "questionee" without adding their own interpretations.
+This way the language of the questioner is *clean* from their own words.
+Here are two most commonly used questions:
+What kind of ‘x’ is that 'x'?
+Is there anything else about ‘x’?
+(where ‘X’ is a word or phrase that the client has used, e.g "What kind of *problem* is that *problem*?", where problem = X)
```

Comment:
I think this section is more clear when referencing back to the example. You can relate 'X' to something concrete.

Reply:
Done